lib/l10n/messages_de.dart

```diff
@@ -0,0 +1,100 @@
+// DO NOT EDIT. This is code generated via package:intl/generate_localized.dart
```

Comment:
please add all `message_*.dart` to your local ignore

Reply:
added